auth/middleware.go

```diff
@@ -35,9 +38,20 @@ func (k *kratosMiddleware) Session(next echo.HandlerFunc) echo.HandlerFunc {
 		if !*session.Active {
 			return c.String(http.StatusUnauthorized, "Unauthorized")
 		}
+
+		if postgrestJWTToken == "" {
```

Comment:
I think this will need to be cached by identity id, with an expiry that matches the token expiry